.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,26 @@
+name: GoReleaser
```

Comment:
For goreleaser binary build to do the right thing in this repo a config like [this](https://github.com/ipni/index-provider/blob/main/.goreleaser.yaml) is needed. Once that's added try running `goreleaser check` to verify the config, and give `goreleaser --rm-dist --snapshot` a shot to make sure binaries are built as you expect them to (written in `dist` directory by default).

I also recommend adding `dist` to `.gitingore`.

Reply:
> does more with less config
he said!

ok, added the config and tested it all out

Reply:
lol